packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -214,12 +214,10 @@ export class CompileTemplate {
       const template = Handlebars.compile(templateContent);
 
       result = template(command.data, {});
-    } catch (e: unknown) {
-      let errorMessage = `Message content could not be generated`;
```

Comment:
@antonjoel82 it always shows this message 😔 Which is a shame, as the message returned from the error is quite informative. 

This error structure does not stand to the conditions of this guard

Reply:
Lets tackle this TS - error - any thing at a different time, when we can give it more attention in an organized way. wdyt?

Reply:
Yep, sounds good for sure!